blues/data/README.md

```diff
@@ -0,0 +1,3 @@
+Files used for unit testing purposes.
+circle.xml is a force field.
```

Comment:
Right, well, I gathered it was a force field, but one might wonder, "what does this have to do with a circle?", or "what force field??" For example, "circle.xml is a simple force field used for unit testing purposes which applies a harmonic potential to (whatever) (something having to do with a circle)," or whatever it is.